### DISCUSSION TITLE
106.0.5249.163 is missing the recent zero-day fix

### DISCUSSION THREAD

**Discussion by SkewedZeppelin** [bromite/bromite#2421] [General]
Bromite 106.0.5249.163: https://chromium.googlesource.com/v8/v8/+log/74c2f33b193963001b885308626be1af3bef14ad
stable LTS 106.0.5249.168: https://chromium.googlesource.com/v8/v8/+log/35a563902756e9277fd9ca903dd08b187fedae78

the former lacks the fix for [CVE-2022-3723](https://chromereleases.googleblog.com/2022/10/stable-channel-update-for-desktop_27.html): https://chromium.googlesource.com/v8/v8/+/ec236fef54b801fcfd89342108cfe95d20db2fd8

granted this shouldn't be too bad considering JIT Is default disabled

**Comment by SkewedZeppelin** [comment]
Bromite [107.0.5304.96](https://github.com/bromite/bromite/commit/a34a56760ffc93efb21a2e2e6005171d18237f7f) was released.
Latest stable is 107.0.5304.105

Here are the differences: https://chromium.googlesource.com/chromium/src/+log/107.0.5304.96..107.0.5304.105?pretty=fuller&n=10000

Of the ten security issues fixed in .105: https://chromereleases.googleblog.com/2022/11/stable-channel-update-for-desktop.html
Bromite/.96 appears to only contains one of the fixes: 1378916

And misses the rest: CVE-2022-3885, CVE-2022-3886, CVE-2022-3887, CVE-2022-3888, CVE-2022-3889, CVE-2022-3890, 1377775, 1375290, 1376639

**Reply by csagan5** [reply]
How do you establish which CVEs correspond to which commits? I see it's a manual process but I wonder how you do it.

**Reply by SkewedZeppelin** [reply]
I just pull the commit log in one window, and the changelog in another window and ctrl+f for the crbug ID of the security fixes.
Every commit will always reference the crbug ID.

I haven't found a better way to do it yet.

**Reply by uazo** [reply]
I confirm, it is the same method I use.
by the way I thought it wasn't so public knowledge... my intention was to automate that process, now that I know others know it too I have no qualms...

**Reply by csagan5** [reply]
Ok, thanks. I thought that perhaps that you had access to those issues.

> my intention was to automate that process, now that I know others know it too I have no qualms...

In #1796 I only implemented the major version update logic, but I might give this a try and theoretically also list the CVEs here: https://www.bromite.org/vulnerable-version

That was the original plan but it would have been better to have access to the Chromium issues because a lot of these do not apply to Android/Bromite and it is just not easy to figure it out.

**Comment by SkewedZeppelin** [comment]
Recently released 108.0.5359.75 is missing the zero day fix for added in 108.0.5359.79.
https://chromereleases.googleblog.com/2022/12/stable-channel-update-for-desktop.html

v8@108.0.5359.75: https://chromium.googlesource.com/v8/v8/+log/18865d6af0404f2d2aeb1c99dd73503364ce0967
v8@108.0.5359.79: https://chromium.googlesource.com/v8/v8/+log/3155b0d10c058d2a9f1d7bba00ad398b3e03b841

Note the missing commit: [[parser] Fix eval tracking](https://chromium.googlesource.com/v8/v8/+/714f34ffdf599a401a9b4389e00fe7def7c16536)
